docs/wiki/Whats-new.md

```diff
@@ -3,22 +3,88 @@
 - [In this Section](#in-this-section)
 - [Updates](#updates)
   - [November 2022](#november-2022)
+    - [Docs](#docs)
+    - [Tooling](#tooling)
+  - [Policy](#policy)
+  - [Other](#other)
   - [October 2022](#october-2022)
+    - [Docs](#docs-1)
+    - [Tooling](#tooling-1)
+  - [Policy](#policy-1)
+  - [Other](#other-1)
   - [September 2022](#september-2022)
+    - [Docs](#docs-2)
+    - [Tooling](#tooling-2)
+  - [Policy](#policy-2)
+  - [Other](#other-2)
   - [August 2022](#august-2022)
+    - [Docs](#docs-3)
+    - [Tooling](#tooling-3)
+  - [Policy](#policy-3)
+  - [Other](#other-3)
   - [July 2022](#july-2022)
+    - [Docs](#docs-4)
+    - [Tooling](#tooling-4)
+  - [Policy](#policy-4)
+  - [Other](#other-4)
   - [June 2022](#june-2022)
+    - [Docs](#docs-5)
+    - [Tooling](#tooling-5)
+  - [Policy](#policy-5)
+  - [Other](#other-5)
```

Comment:
@lachaves please roll all these indents back. Likely was an extension in VSCode.

We only want the update to content, not the TOC.